var/spack/repos/builtin/packages/py-py2cairo/package.py

```diff
@@ -35,7 +35,7 @@ class PyPy2cairo(Package):
 
     extends('python')
 
-    depends_on('cairo+X')
+    depends_on('cairo')
```

Comment:
I'm actually the one who added `+X` to this dependency. When I was working on #2778, I noticed that this package did not build for me because it was looking for X11 dependencies. Adding `+X` solved the problem for me.